src/db.js

```diff
@@ -6,7 +6,10 @@
  *
  */
 
-import {Pool} from 'react-pg';
+// Error early if this is accidentally imported on the client.
+import 'server-only';
```

Comment:
Add server-only to credentials.js?

Reply:
Maybe both? I was thinking it’s too easy to miss outside src folder.

Reply:
I think they’re also imported from a seed script.

Reply:
Both is fine but it seems like the whole point is you stick it in your secret file.